activerecord/test/cases/arel/attributes/math_test.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+require_relative "../helper"
+
+module Arel
+  module Attributes
+    class MathTest < Arel::Spec
+      %i[* /].each do |math_operator|
+        it "average should be compatiable with #{math_operator}" do
+          table = Arel::Table.new :users
+          (table[:id].average.public_send(math_operator, 2)).to_sql.must_be_like %{
+            AVG("users"."id") #{math_operator} 2
+          }
+        end
+
+        it "count should be compatiable with #{math_operator}" do
+          table = Arel::Table.new :users
+          (table[:id].count.public_send(math_operator, 2)).to_sql.must_be_like %{
+            COUNT("users"."id") #{math_operator} 2
+          }
+        end
+
+        it "maximum should be compatiable with #{math_operator}" do
+          table = Arel::Table.new :users
+          (table[:id].maximum.public_send(math_operator, 2)).to_sql.must_be_like %{
+            MAX("users"."id") #{math_operator} 2
+          }
+        end
+
+        it "minimum should be compatiable with #{math_operator}" do
+          table = Arel::Table.new :users
+          (table[:id].minimum.public_send(math_operator, 2)).to_sql.must_be_like %{
+            MIN("users"."id") #{math_operator} 2
+          }
+        end
+
+        it "table node should be compatiable with #{math_operator}" do
```

Comment:
`table[:id]` is not table node, but attribute node.

https://github.com/rails/rails/blob/df850b8d7126044a820291bbfd2bce80636a442c/activerecord/lib/arel/table.rb#L81-L83

https://github.com/rails/rails/blob/df850b8d7126044a820291bbfd2bce80636a442c/activerecord/lib/arel/attributes/attribute.rb#L5-L10

Reply:
Thank you, I've updated it.